src/main/java/org/opensearch/security/filter/SecurityFilter.java

```diff
@@ -507,6 +511,31 @@ private void attachSourceFieldContext(ActionRequest request) {
         }
     }
 
+    private <Response extends ActionResponse> boolean handlePermissionCheckRequest(
+        ActionListener<Response> listener,
+        PrivilegesEvaluatorResponse pres
+    ) {
+        String isSimulation = threadContext.getHeader(ConfigConstants.SECURITY_HAS_PERMISSION_CHECK_PARAM);
+        if (Boolean.parseBoolean(isSimulation)) {
+
+            try {
+
+                @SuppressWarnings("unchecked")
+                Response response = (Response) new PermissionCheckResponse(pres.isAllowed(), pres.getMissingPrivileges());
+                listener.onResponse(response);
+
+                log.info("accessAllowed {}", pres.isAllowed());
```

Comment:
these logs should be changed to debug level and should contain information about what action was simulation run on.